app/Domain/ResumeGenerator.php

```diff
@@ -24,7 +24,10 @@ public function generate(Resume $resume): string
         $this->fillEducation($processor, $resume);
         $this->fillProjects($processor, $resume);
 
-        return $processor->save();
+        $path = storage_path("app/resumes/" . now() . ".docx");
```

Comment:
Which date format will be used here? How path will looks like?

Reply:
its temp file, we are deleting them after download